typings/connection.d.ts

```diff
@@ -505,7 +505,7 @@ export interface MessageHeader {
   /**
    * @property {number} [ttl] time to live in ms.
    */
-  ttl?: number;
+  ttl?: number | Date;
```

Comment:
I don't think TTL should be a Date. It is not encoded as a timestamp but as a uint representing the number of milliseconds.

Reply:
Makes sense. I might have been a bit overzealous with my demo there. :|

Reply:
(removed this in latest push)